src/databases/Curve3D/avtCurve3DFileFormat.C

```diff
@@ -0,0 +1,1122 @@
+// Copyright (c) Lawrence Livermore National Security, LLC and other VisIt
+// Project developers.  See the top-level LICENSE file for dates and other
+// details.  No copyright assignment is required to contribute to VisIt.
+
+// ************************************************************************* //
+//                            avtCurve3DFileFormat.C                         //
+// A reader for reading files that contain 3D curves. These may be named using
+// the same convention as in Curve2D, or not.
+// White space can be considered as a segmentation when MULTSEG is set.
+// In that case, we get the same behaviour as the one of gnuplot with the splot
+// function.
+// 2023/10/05 Olivier Cessenat.
+// Verbosity for poor man's debugging
+#define INTERACTIVEPLOT 0
+// Consider white space indicates a split
+#define MULTSEG 1
```

Comment:
Would it make sense to make this a run-time `DBOptionsAttributes` as in, for example, https://github.com/visit-dav/visit/blob/7f7c4f6ed8e177b06700b412ab8a3630241341b9/src/databases/BATL/avtBATLFileFormat.C#L191-L192

Reply:
Yes, good idea. As a default I will validate segmentation.
I left the MULTSEG 1 for coding check : when MULTSEG is set to 1, the option is available.